recipes/newrelic/apm/dotNet/windows-iis.yml

```diff
@@ -25,6 +25,8 @@ preInstall:
     $iisEnabled = Get-WindowsOptionalFeature -Online -FeatureName IIS-WebServer | select State
     if ($iisEnabled.State -ne "Enabled") {
       # IIS is not installed
+      Write-Host "Required dependency (IIS) is not installed. To enable, run:"
```

Comment:
I don't know if customers can actually see this message.
@Julien4218 Can they see this?

Reply:
@jaffinito no this message would not be seen by anyone. Script in the "preInstall" are only used for evaluating whether the recipe should be run, so it's a yes (0 exit code) or false (non 0 exit code) answer.
Any output is discarded.

Reply:
I'll pull these logging changes out